test/suites/integration/bsp/slash-provider.test.ts

```diff
@@ -1,65 +1,69 @@
 import { strictEqual } from "node:assert";
 import { describeBspNet, type EnrichedBspApi } from "../../../util";
 
-describeBspNet("BSPNet: Slash Provider", ({ before, createUserApi, createBspApi, it }) => {
-  let userApi: EnrichedBspApi;
-  let bspApi: EnrichedBspApi;
+describeBspNet(
+  "BSPNet: Slash Provider",
+  { only: true },
```

Comment:
Are these `only` leftover from debugging?

Reply:
yes thanks for spotting